Fill changelog for v0.13

adonath opened the pull request:
This PR fills the changelog for Gammapy v0.13. I've summarised the most important bug fixes, API changes and new features.

Please let me know, if you have any comment or suggestions for improvement @cdeil, @registerrier, @AtreyeeS, @luca-giunti, @Bultako, @fabiopintore, @facero, @yuwunwong, @kaoriinakashima, @bsipocz, @bkhelifi, @manconi, @JouvinLea and @QRemy.

cdeil commented on CHANGES.rst:
```diff
@@ -1,14 +1,128 @@
+class. We also removed the `Energy` and `EnergyBounds` classes. Grids of energies
+can be created and handled directly now using the `MapAxis` object.
+
+We added a new `SpectralGaussian` and `PLSuperExpCutoff4FG` model and changed
```
Typo: 4FGL

adonath replied:
Fixed

cdeil commented on CHANGES.rst:
```diff
@@ -1,14 +1,128 @@
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+
+- [#2310] Remove old `LightCurveEstimator` class (Axel Donath)
```
If you can wait an hour, please list https://github.com/gammapy/gammapy/pull/2296, I'd like to merge it in today.

adonath replied:
Yes, we will wait. I added it to the changelog...

adonath commented:
I changed to a less chatty style, changed the order of the paragraphs and added one sentence about the `LightCurveEstimator`. If you have additional comments, please submit them before 16h00 today.

bsipocz commented on CHANGES.rst:
```diff
@@ -1,14 +1,127 @@
-run and give the same outputs.
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+- [#2296] Implement model YAML serialisation (Quentin Remy)
```
I'm not a particularly big fan of this type of structuring, as it's very difficult to get any useful info out of it when one looks for reasons of a breaking piece of code in certain modules. This current layout can be easily obtained by clicking on the milestone on github and browsing the title of the PRs.

Examples for a more useful structure (imo) are e.g. astropy, scikit-learn, or even mpl.

adonath replied:
That is the structure we have been using so far in Gammapy, but I agree there are better solutions. For this release it probably comes too late, but we might consider a different structure for future releases.

cdeil replied:
@bsipocz - Agreed there are better ways to write what's new and changelogs.
@adonath - Also agree to just get this out today, and then to do better with future releases.

adonath commented on CHANGES.rst:
```diff
@@ -1,14 +1,127 @@
+The Gammapy v0.13 release includes many bug-fixes, a lot of clean-up work
+and some new features.
+
+Gammapy v0.13 implements a new `SpectralGaussian` and `PLSuperExpCutoff4FGL`
```
Fixed

adonath commented on CHANGES.rst:
```diff
@@ -1,14 +1,127 @@
+
+- Released on July 26th, 2019
+- 15 contributors
+- 2 months of work
```
Yes, it refers to the 2 month release cycle.

adonath commented on CHANGES.rst:
```diff
@@ -1,14 +1,127 @@
+interface, improvements in data reduction, event sampling and time analysis.
+We already started the implementation of YAML serialisation for models, event
+sampling and better lightcurve estimation. Stay tuned for the next Gammapy
+release in September!
```
Good point, I decided to remove this paragraph again. It was probably inspired too much from changelogs you often read in App-stores ;-)

adonath commented:
Thanks for your feedback @bsipocz!
